src/main/java/de/dennisguse/opentracks/util/EGM2008Utils.java

```diff
@@ -65,7 +65,7 @@ static Indices getIndices(Location location) {
         if (location.getLongitude() >= 0) {
             longitude = location.getLongitude();
         } else {
-            longitude = 180 + Math.abs(location.getLongitude());
+            longitude = 360 + location.getLongitude();
         }
         int longitudeIndex = (int) (longitude * RESOLUTION_IN_MINUTES);
```

Comment:
Now I follow and this seems ok.  (comment in PR was earlier).  I do think this is needlessly hard to follow because of the if structure, rather than
  get longtitude
  if longitude is negative, += 360 with comment that we are doing range normalization from (-180,180] to [0,360)